src/SDKs/DevSpaces/Management.DevSpaces/Microsoft.Azure.Management.DevSpaces.csproj

```diff
@@ -6,7 +6,7 @@
     <PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DevSpaces</PackageId>
 		<Description>Provides management operations support for Azure Dev Spaces.</Description>
-		<Version>0.10.0-preview</Version>
+		<Version>0.11.0-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.DevSpaces</AssemblyName>
 		<PackageTags>Microsoft Azure Dev Spaces;Dev Spaces;</PackageTags>
         <PackageReleaseNotes>Taking dependency on 10.0.3 version of Newtonsoft nuget package.</PackageReleaseNotes>
```

Comment:
Please update the `PackageReleaseNotes` here

Reply:
Done